instrumentation/rdkafka/lib/opentelemetry/instrumentation/rdkafka/patches/consumer.rb

```diff
@@ -74,6 +74,14 @@ def extract_message_key(key)
           rescue Encoding::UndefinedConversionError
             nil
           end
+
+          def getter
+            if Gem::Version.new(::Rdkafka::VERSION) >= Gem::Version.new('0.13.0')
```

Comment:
Is there a way to check/set this only once when the gem is activated as opposed to each time a message is processed?

Reply:
we could memoize like 661cd674e1668b62092a25239c44c5b2c4ddecff or do you think it's better to use a constant like this?
```ruby
          GETTER = if Gem::Version.new(::Rdkafka::VERSION) >= Gem::Version.new('0.13.0')
              Context::Propagation.text_map_getter
            else
              OpenTelemetry::Common::Propagation.symbol_key_getter
            end
          private_constant :GETTER
```

(btw we still need to support rdkafka `0.12.0`, right?)

Reply:
fyi, ` 0.12.0` is released on June 17, 2022, and the latest version of rdkafka is `0.15.1` as of now

https://rubygems.org/gems/rdkafka/versions